src/main/statistics_gathering/statistics_gathering.py

```diff
@@ -108,13 +108,24 @@ def get_statistics(path: str):
 
 # run after 'split_tasks_into_separate_files' to print simple statistic
 def get_tasks_statistic(path: str):
+    statistic = {}
     languages = [l.value for l in consts.LANGUAGE]
     language_folders = get_all_file_system_items(path, (lambda f: does_string_contain_any_of_substrings(f, languages)),
                                                  consts.FILE_SYSTEM_ITEM.SUBDIR.value)
     for l_f in language_folders:
-        print(get_name_from_path(l_f, False))
+        language = consts.LANGUAGE(get_name_from_path(l_f, False))
+        if statistic.get(language) is not None:
```

Comment:
There is another way to check if variable is not None:
`if not statistic.get(language)`
instead of
`if statistic.get(language) is not None:`
because of python typecast to boolean

For me, the first way is more readable because the second has 'double negative': not and None. However, you can leave it as you prefer, I just want you to be aware of the other option.

Reply:
Really better, thanks!